src/main/java/com/dakar/dakar/controller/GraphQlController.java

```diff
@@ -50,11 +61,21 @@
         return route(RequestPredicates.POST("/graphql"), request -> {
             if (request.headers().contentType().filter(mediaType -> mediaType.isCompatibleWith(GraphQLMediaType)).isPresent()) {
                 return request.bodyToMono(GraphQLParameter.class)
-                        .flatMap(graphQLParameter -> {
-                            ExecutionInput.Builder executionInput = newExecutionInput()
-                                    .query(graphQLParameter.getQuery())
-                                    .operationName(graphQLParameter.getOperationName())
-                                    .variables(graphQLParameter.getVariables());
+                        .flatMap((GraphQLParameter graphQLParameter) -> {
+                            String errorMessage = graphQLValidator.validateGraphQL(graphQLParameter);
+                            ExecutionInput.Builder executionInput;
+                            if(errorMessage != null && !"".equals(errorMessage)){
```

Comment:
you can use some StringUtils.isNotBlank(errorMessage)

Reply:
That's true, thx